pkg/cloudprovider/providers/azure/azure_instances.go

```diff
@@ -87,10 +86,23 @@ func (az *Cloud) ExternalID(name types.NodeName) (string, error) {
 	return az.InstanceID(name)
 }
 
-// InstanceExistsByProviderID returns true if the instance with the given provider id still exists and is running.
+// InstanceExistsByProviderID returns true if the instance with the given provider id still exists.
```

Comment:
The `az.InstanceID` function says it returns `InstanceNotFound` if the instance is not running.  So perhaps the "and is running" part of the documentation needs to be reinstated?

Reply:
Right, thank you.